src/server/server.py

```diff
@@ -2310,12 +2310,12 @@ def check_bpillfw_file(data):
             if rStr:
                 fwTimStr += " " + rStr
         else:
-            fwTimStr = "unknown"
+            fwTimStr = __("unknown")
         fileSize = len(dataStr)
         logger.debug("Node update firmware file size={}, version={}, {}build timestamp: {}".\
                      format(fileSize, fwVerStr, fwTypStr, fwTimStr))
-        infoStr = "Firmware update file size = {}<br>".format(fileSize) + \
-                  "Firmware update version: {} ({}Build timestamp: {})<br><br>".\
+        infoStr = __("Firmware update file size = {} bytes<br>").format(fileSize) + \
+                  __("Firmware update version: {} ({}Build timestamp: {})<br><br>").\
                   format(fwVerStr, fwTypStr, fwTimStr)

```

Comment:
The non-text parts need to be separated; for instance like this:
```
       infoStr = "{} = {} {}<br>".format(__("Firmware update file size"), fileSize, __("bytes")) + \
                  {}: {} ({}{}: {})<br><br>").\
                  format(__("Firmware update version"), fwVerStr, fwTypStr, __("Build timestamp"), fwTimStr)
```
Similar for several places below.